cmd/soci-snapshotter-grpc/main.go

```diff
@@ -95,6 +96,7 @@ var (
 	logLevel     = flag.String("log-level", defaultLogLevel.String(), "set the logging level [trace, debug, info, warn, error, fatal, panic]")
 	rootDir      = flag.String("root", config.DefaultSociSnapshotterRootPath, "path to the root directory for this snapshotter")
 	printVersion = flag.Bool("version", false, "print the version")
+	dumpConfig   = flag.Bool("config-dump", false, "print contents of resolved configuration")
```

Comment:
Can this wait until after #573 so that we can have a proper `config dump` command?

Reply:
Sounds good to me